nix/myscripts/default.nix

```diff
@@ -19,18 +20,30 @@ in {
         type = lib.types.str;
       };
     };
+    new-lilypond-project.enable = lib.mkEnableOption "new-lilypond-project";
+    randomcase.enable = lib.mkEnableOption "randomcase";
+    we.enable = lib.mkEnableOption "we";
   };
-  config = lib.mkIf config.myscripts.enableAll {
-    home.packages =
-      [
-        (pkgs.callPackage ./we.nix {mpv = config.programs.mpv.finalPackage;})
-        (pkgs.callPackage ./ccopy.nix {})
-        (pkgs.callPackage ./cedit.nix {})
-        (pkgs.callPackage ./mount.nix {})
-        (pkgs.callPackage ./randomcase.nix {})
-        (pkgs.callPackage ./new-lilypond-project.nix {useGh = true;})
-        # (pkgs.callPackage ./mount-go.nix {})
-      ]
-      ++ lib.lists.optional cfg.mrandr.enable (pkgs.callPackage ./mrandr.nix {inherit (cfg.mrandr) OUTPUT SCREEN;});
-  };
+
+  config = lib.mkMerge [
+    (lib.mkIf cfg.enableAll {
+      myscripts = {
+        ccopy.enable = lib.mkDefault true;
+        cedit.enable = lib.mkDefault true;
+        mount.enable = lib.mkDefault true;
+        mrandr.enable = lib.mkDefault true;
+        new-lilypond-project.enable = lib.mkDefault true;
+        randomcase.enable = lib.mkDefault true;
+        we.enable = lib.mkDefault true;
+      };
+    })
+
+    (lib.mkIf cfg.ccopy.enable {home.packages = [(pkgs.callPackage ./ccopy.nix {})];})
+    (lib.mkIf cfg.cedit.enable {home.packages = [(pkgs.callPackage ./cedit.nix {})];})
+    (lib.mkIf cfg.mount.enable {home.packages = [(pkgs.callPackage ./mount.nix {})];})
+    (lib.mkIf cfg.mrandr.enable {home.packages = [(pkgs.callPackage ./mrandr.nix {inherit (cfg.mrandr) OUTPUT SCREEN;})];})
+    (lib.mkIf cfg.new-lilypond-project.enable {home.packages = [(pkgs.callPackage ./new-lilypond-project.nix {useGh = true;})];})
+    (lib.mkIf cfg.randomcase.enable {home.packages = [(pkgs.callPackage ./randomcase.nix {})];})
+    (lib.mkIf cfg.we.enable {home.packages = [(pkgs.callPackage ./we.nix {})];})
```

Comment:
> Any other suggestions in terms of formatting and organizing the code here?

Wrap the code at 80 characters and reduce indentation nesting with `lib.singleton`:

```nix
(
  lib.mkIf cfg.<OPTION_1>.enable {
    home.packages = lib.singleton (pkgs.callPackage <FILE_1> {});
  }
)

(
  lib.mkIf cfg.<OPTION_2>.enable {
    home.packages = lib.singleton (
      pkgs.callPackage <FILE_2> {
        inherit pkgs;

        one = 1;
        two = 2;
      }
    );
  }
)
```